Plugins/Geant4/src/Geant4Converters.cpp

```diff
@@ -319,9 +319,11 @@ Acts::Geant4ShapeConverter::planarBounds(const G4VSolid& g4Solid) {
 namespace {
 Acts::Transform3 axesOriented(const Acts::Transform3& toGlobalOriginal,
                               const std::array<int, 2u>& axes) {
-  auto originalRotation = toGlobalOriginal.rotation();
-  auto colX = originalRotation.col(std::abs(axes[0u]));
-  auto colY = originalRotation.col(std::abs(axes[1u]));
+  // auto originalRotation = toGlobalOriginal.rotation();
```

Comment:
```suggestion
```

Also, this shouldn't matter with the change, no?

Reply:
Shouldn't matter but I kept linear. Removed the comment.